flow/plasma_vx.x.x.js

```diff
@@ -222,24 +220,26 @@ declare module '@wework-dev/plasma' {
   |}> {}
 
   // source file: src/components/NumberInput/NumberInput.jsx
-  declare type HandleEvent = {|
-    nativeEvent: { target: { value: any } },
-    target: { parentElement: { classList: { add: string => mixed, remove: string => mixed } } },
-  |};
+  declare type HandleEvent =
+    | {|
+        nativeEvent: { target: { value: any } },
+        target: { parentElement: { classList: { add: string => mixed, remove: string => mixed } } },
+      |}
+    | string;
```

Comment:
is this correct? when would HandleEvent be a string?

Reply:
We are passing it a string in one place in plasma it seems

I can look more tomorrow when I am by my computer